docs/website/docs/dlt-ecosystem/destinations/filesystem.md

```diff
@@ -220,11 +220,63 @@ Current default layout: **{table_name}/{load_id}.{file_id}.{ext}`**
 > 💡 Note that the default layout format has changed from `{schema_name}.{table_name}.{load_id}.{file_id}.{ext}` to `{table_name}/{load_id}.{file_id}.{ext}` in dlt 0.3.12. You can revert to the old layout by setting the old value in your toml file.
 
 
+### Available layout placeholders
+
+#### Standard placeholders
+
+* `schema_name` - the name of schema
+* `table_name` - table name
+* `load_id` - the id of the load package form which the file comes from
+* `file_id` - the id of the file
+* `ext` - a format of the file i.e. jsonl or parquet
+
+#### Date and time placeholders
+> 💡 Keep in mind all values are lowercased.
+
+* Years
+  * `YYYY` - 2024, 2025
+  * `Y` - 2024, 2025
+* Months
+  * `MMMM` - January, February, March
+  * `MMM` - Jan, Feb, Mar
+  * `MM` - 01, 02, 03
+  * `M` - 1, 2, 3
+* Days of month
+  * `DD` - 01, 02
+  * `D` - 1, 2
+* Hours 24h format
+  * `HH` - 00, 01, 02...23
+  * `H` - 0, 1, 2...23
+* Minutes
+  * `mm` - 00, 01, 02...59
```

Comment:
What timestamp would be used for minutes extraction? `timestamp` or `load_package_timestamp`?

Reply:
The logic is following 
* If `current_datetime` argument is given
* Then we take from it,
* Else take `current_datetime=load_package_timestamp`

Reply:
`load_package_timestamp` is always taken from current load package

Reply:
Do you mean `current_datetime` argument should be passed in pipeline? And it would be the same for all files then?

```
pipeline = dlt.pipeline(
    pipeline_name="data_things",
    destination=filesystem(
        layout = "{table_name}/{placeholer_x}/{timestamp}/{load_id}.{file_id}.{ext}",
        current_datetime=datetime.now(),
        # current_datetime=get_current_datetime,
        extra_placehodlers={
            "test_placeholder": "test",
            "placeholder_x": placeholder_callback
        }
    )
)
```

Reply:
yes you can pass it and it will the same for all files if used